src/System Application/App/Upgrade Tags/src/UpgradeTag.Codeunit.al

```diff
@@ -43,6 +43,16 @@ codeunit 9999 "Upgrade Tag"
         exit(UpgradeTagImpl.HasUpgradeTag(Tag, TagCompanyName));
     end;
 
+    /// <summary>
+    /// Verifies if the database upgrade tag exists.
+    /// </summary>
+    /// <param name="Tag">Tag code to check</param>
```

Comment:
Could you add a comment that the developer can use HasUpgradeTag if it is called from the upgrade context? If it is called from upgrade context e.g. OnUpgradePerDatabase we will automatically use perdatabase check and similar for percompany? 

Nothing wrong with using this one directly but it could be confusing if someone is reading the existing code.

Reply:
Seems to be a minor. Can be addressed in a chaser pack.